Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM node:15 AS verify-format
+WORKDIR /src
+COPY package.json yarn.lock ./
+RUN yarn
+COPY . .
+RUN yarn verify-format
+
+FROM koalaman/shellcheck-alpine as verify-sh
+WORKDIR /src
+COPY ./*.sh ./
+RUN shellcheck -e SC1091,SC1090 ./*.sh
+
+FROM mcr.microsoft.com/dotnet/sdk:5.0-buster-slim AS restore
```

Comment:
```suggestion
FROM mcr.microsoft.com/dotnet/sdk:5.0 AS restore
```

Reply:
Thanks @elsupergomez, what are the differences between them? why one option is better than the other one?

Reply:
We talked with @elsupergomez about this. I think that this image is a multi-platform one, so, you can base on this one to build for Windows or Linux. 

In any case, is the default image used in Visual Studio, so, I am fine with it.